ROMFS/px4fmu_common/init.d-posix/airframes/6011_gazebo-classic_typhoon_h480.post

```diff
@@ -1,8 +1,6 @@
 
 mavlink start -x -u 14558 -r 4000 -f -m onboard -o 14530 -p
 
-# shellcheck disable=SC2154
-mavlink stream -r 10 -s MOUNT_ORIENTATION -u $udp_gcs_port_local
 # shellcheck disable=SC2154
 mavlink stream -r 50 -s ATTITUDE_QUATERNION -u $udp_offboard_port_local
 mavlink stream -r 10 -s MOUNT_ORIENTATION -u $udp_offboard_port_local
```

Comment:
@frederictaillandier @julianoes we still have it on this line. Shouldn't we remove it here as well?

Reply:
It solves my issue for me, but not sure if it breaks anything for anyone :D I will make a PR in a week or 2 just to be sure that nobody complains about this PR, and then eventually I would even remove $udp_gcs_port_local everywhere